assets/src/edit-story/components/canvas/pagepreview/index.js

```diff
@@ -73,7 +74,7 @@ const PreviewWrapper = styled.div`
   background-position: 0 0, 0 4px, 4px -4px, -4px 0px;
 `;
 
-function PagePreview({ index, forwardedRef, ...props }) {
+const PagePreview = React.forwardRef(({ index, ...props }, forwardedRef) => {
```

Comment:
Is `forwaredRef` still needed here at all? It seemed like a product of DND originally.